frontend/www/js/omegaup/components/problem/FilterTags.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="card">
+    <div class="card-body">
+      <h3>{{ T.problemEditAddTags }}</h3>
+      <div v-for="(tag, index) in tags" :key="index" class="form-check">
+        <label class="form-check-label">
+          <input
+            v-model="localSelectedTags"
+            :value="tag"
+            class="form-check-input"
+            type="checkbox"
+          />{{ T[tag] }}
+        </label>
+      </div>
+      <div class="form-group">
+        <vue-typeahead-bootstrap
+          :data="publicTags"
+          :serializer="publicTagsSerializer"
+          :placeholder="T.collecionOtherTags"
+          @hit="addOtherTag"
+        >
+        </vue-typeahead-bootstrap>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
+import T from '../../lang';
+import VueTypeaheadBootstrap from 'vue-typeahead-bootstrap';
+
+@Component({
+  components: {
+    'vue-typeahead-bootstrap': VueTypeaheadBootstrap,
+  },
+})
+export default class CollectionFilterTags extends Vue {
+  @Prop() tags!: string[];
+  @Prop() publicTags!: string[];
+  @Prop({ default: () => [] }) selectedTags!: string[];
+
+  T = T;
+  localSelectedTags = this.selectedTags;
```

Comment:
Acá siempre se iniciará sin `checkbox` seleccionados, ¿verdad?, de ser así, esto puede quedar así:
```suggestion
  selectedTags: string[] = [];
```
También, como ya sólo se manejaran desde aquí los tags seleccionados, puedes cambiarle el nombre.